src/System.Private.CoreLib/Common/System/SR.cs

```diff
@@ -44,11 +45,11 @@ internal static string GetResourceString(string resourceKey, string defaultStrin
                 return defaultString;
             }
 
-            return resourceString;
+            return resourceString!; // only null if missing resource
```

Comment:
I guess you did this because it is an internal API and we don't want to add a bunch of `!` everywhere?

Reply:
And resources should technically always be present.

Reply:
> And resources should technically always be present.

Right; if they're not, there's a larger issue there to be addressed.